docs/api/apiv3/paths/days_non_working_date.yml

```diff
@@ -0,0 +1,133 @@
+      schema:
+        type: string
+        format: date
+      example: '2022-05-06'
+  responses:
+    '200':
+      description: OK
+      content:
+        application/hal+json:
+          schema:
+            $ref: '../components/schemas/non_working_day_model.yml'
+    '400':
+      description: Returned if the client sends invalid request parameters.
+      content:
+        application/hal+json:
+          schema:
+            $ref: '../components/schemas/error_response.yml'
+    '404':
+      description: Returned if the given date is not a non-working day.
+      content:
+        application/hal+json:
+          schema:
+            $ref: '../components/schemas/error_response.yml'
+          example:
+            _type: Error
+            errorIdentifier: urn:openproject-org:api:v3:errors:NotFound
+            message: The requested resource could not be found.
+
+# TODO: add missing description, missing tags, missing parameters, and missing response codes
+patch:
+  summary: Update a non-working day attributes
+  operationId: update_non_working_day
+  tags:
+    - Work Schedule
+    - Non working days
+  description: |-
+    Update the non-working day information for a given date.
+  parameters:
+    - name: date
+      in: path
+      required: true
+      description: |-
+        The date of the non-working day to view in ISO 8601 format.
+      schema:
+        type: string
+        format: date
+      example: '2022-05-06'
+  requestBody:
+    content:
+      application/json:
+        schema:
+          $ref: ../components/schemas/non_working_day_model.yml
+        example:
+          _type: NonWorkingDay
+          date: '2022-05-01'
+          name: 'Labour day'
+
+  responses:
+    '200':
+      description: OK
+      content:
+        application/hal+json:
+          schema:
+            $ref: '../components/schemas/non_working_day_model.yml'
+    '400':
+      description: Returned if the client sends invalid request parameters.
+      content:
+        application/hal+json:
+          schema:
+            $ref: '../components/schemas/error_response.yml'
+    '404':
+      description: Returned if the given date is not a non-working day.
+      content:
+        application/hal+json:
+          schema:
+            $ref: '../components/schemas/error_response.yml'
+          example:
+            _type: Error
+            errorIdentifier: urn:openproject-org:api:v3:errors:NotFound
+            message: The requested resource could not be found.
+
+# TODO: add missing description, missing tags, missing parameters, and missing response codes
+delete:
+  summary: Removes a day from the non-working days
+  operationId: delete_non_working_day
+  tags:
+    - Work Schedule
+    - Non working days
+  description: |-
+    Removes the day of the given day from the non-working days known.
+
+    Trying to remove a date which is not a non-working day, by calling the same
```

Comment:
I thought it would be better but [I'm probably wrong](https://stackoverflow.com/questions/6439416/status-code-when-deleting-a-resource-using-http-delete-for-the-second-time), and returning 404 would probably be better.

Reply:
Fixed in 54ee9fc31b